users/config.rst

```diff
@@ -81,6 +81,7 @@ The following shows an example of the default configuration file (IDs will diffe
             <maxSendKbps>0</maxSendKbps>
             <maxRecvKbps>0</maxRecvKbps>
             <maxRequestKiB>0</maxRequestKiB>
+            <remoteGUIPort>8384</remoteGUIPort>
```

Comment:
The value should be

```
<remoteGUIPort>0</remoteGUIPort>
```

here, as this is supposed to show the default configuration. The default value is `0`.